docs/build/guides/archival/extend-persistent-entry-js.mdx

```diff
@@ -0,0 +1,179 @@
+---
+title: Extend a persistent entry and contract using the JavaScript SDK
+description: How to extend a persistent entry and contract using the JavaScript SDK
+---
+
+Persistent storage is used for storing data and contracts that need to remain on the network for a long period.
```

Comment:
Contract instances and contract code(wasm hash) are actually instance storage with persistent durability